style.css

```diff
@@ -0,0 +1,32 @@
+.block {
+    margin-left: auto;
+    margin-right: auto;
+    width: 200px
+}
+.red:hover, .green:hover, .blue:hover, .yellow:hover, .orange:hover {
+    height: 30vh;
+}
+.red {
+    height: 20vh;
+    background: red;
+}
+.green {
+    height: 20vh;
+    background: green;
+}
+.blue {
+    height: 20vh;
+    background: blue;
+}
+.yellow {
+    height: 20vh;
+    background: yellow;
+}
+.orange {
+    height: 20vh;
+    background: orange;
+}
+* {
```

Comment:
all five classes has the same properties, when you will add common class, you can add this properties only one time, the same with hover effect